spec/lib/stats_reporter_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'stats_reporter'
+require File.expand_path(File.dirname(__FILE__) + '../../../config/boot')
+describe StatsReporter do
+
+  describe '.df_output' do
+    it 'outputs stats on disk space' do
+      output = described_class.df_output
+      # for example, this matches '5.0G  2.1G  2.6G  45% /'
+      size_regex = '[0-9]+(\.[0-9]+)?[A-Za-z]'
+      percent_regex = '[0-9]+\%'
+      regex = %r{#{size_regex}.+#{size_regex}.+#{size_regex}.+#{percent_regex}.*\s\/}
+      expect(output).to match(regex)
+    end
+  end
+
```

Comment:
my question about these table specs:  do we want to check the exact table, or check for the table header and content?   Okay leaving it the way it is for now ...

Reply:
I'm OK with checking the exact table, but I'm willing to be convinced we should do otherwise.